File: Display_image_and_video/Display_image.cpp
#include <opencv2/opencv.hpp>
#include <iostream>

using namespace std;
using namespace cv;

int main(int argc,char **argv)
{
	//To read the image from the disk................
	Mat img = imread(argv[1],1);  //argv[1]=input image
	if(img.empty())
	{
		cout<<"Error,image can't be read"<<endl;
	}

	//To display the number of channels
	cout<<"Channels="<<img.channels()<<endl;

	//To display the size of the image
	cout<<"Height = "<<img.rows<<endl;
	cout<<"Width = "<<img.cols<<endl;
	cout<<"Size = "<<img.size<<endl;
	
	//To display the image with normal window size
	namedWindow("IMSHOW",WINDOW_NORMAL);
	imshow("IMSHOW",img);
	imwrite("original.jpg",img);
	waitKey(0);
	

}

File: Display_image_and_video/Display_image.py
import cv2
import sys

#Reading the image and image path will take from the command line
image = cv2.imread(sys.argv[1],1)

if image.any():
	print("Read the image successfully.............")
else:
	print("Error , Please provide the correct image")

#prints the size and channels information.............
print("Size of the image : ",image.shape)
cv2.namedWindow("Display_image",cv2.WINDOW_NORMAL)
cv2.imshow("Display_image",image)
cv2.waitKey(0)